src/vorta/views/repo_tab.py

```diff
@@ -147,7 +147,12 @@ def init_repo_stats(self):
             self.repoEncryption.setText(str(repo.encryption))
         else:
             # Compression and SSH key are only valid entries for a repo
-            self.frameRepoSettings.setEnabled(False)
+            # Yet Add SSH key button must be enabled for bootstrapping
+            self.frameRepoSettings.setEnabled(True)
+            for child in self.frameRepoSettings.children():
+                if not isinstance(child, QLayout):
+                    child.setEnabled(False)
```

Comment:
The children are disable, but not enabled again if the repo is set.

Reply:
Indeed. Fixed in a new revision.